feat: Create sig-security (#116)
argoproj/argoproj: sigs/sig-security/README.md

> # Argo Sig Security

Reviewer: ```suggestion
# Argo SIG Security
```

> Argo Sig Security is focused on improving security across the Argo project, promoting best practices, and ensuring a high level of security for end users.

Reviewer: ```suggestion
Argo SIG Security is focused on improving security across the Argo project, promoting best practices, and ensuring a high level of security for end users.
```
I think we should capitalize SIG for being an abbreviation.

## Meetings

> Sig security meetings include public and non-public sections and are held twice monthly. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com&ctz=America%2FDenver).

Reviewer: ```suggestion
SIG security meetings include public and non-public sections and are held twice monthly. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com&ctz=America%2FDenver).
```

## Security Policy and Reporting Issues

> Argo Project security policy and instructions for reporting issues is availabile [here](https://github.com/argoproj/argoproj/blob/master/SECURITY.md).

Reviewer: ```suggestion
Argo Project security policy and instructions for reporting issues are available [here](https://github.com/argoproj/argoproj/blob/master/SECURITY.md).
```

> 

Reviewer 2: ```suggestion
## Membership Requirements
- Must be an Argo project maintainer from the current list (https://github.com/argoproj/argoproj/blob/master/MAINTAINERS.md).
- Must have shown competency in discovering, solving, and communicating in the project security context.
- Must have shown a proven record of solid coding development.
- Must represent the open-source community’s best interest with a vendor-neutral practice.
- The membership will be revisited every calendar year.
```

Reviewer 2: The team agrees on the maintainer requirement.

Reviewer 2: Line 13/14: replace to "The maintainers to decide the membership." "Define a process for adding/removing."

> ## Members

Reviewer 2: I recommend keeping the members as the maintainers and also engineers. I haven't seen Sasha once in any Argo meetings. I am hesitant to agree to grant the privileges and membership here.

Reviewer 2: To be clear and comprehensive, no membership doesn't prevent interested individuals and parties to participate in the discussion, putting effort, resources. However, I feel only the contributors are more entitled to the membership as it could be translated to make/represent some key decisions for the communities, and those people who are doing the day-to-day coding, should be trusted more in the security context.

Author: Sasha has joined several of the sig-security meetings and contributed there.

Reviewer: I tend to agree with the concerns of @wanghong230. However, I also think it's great to have someone with a more external (and specialized) point of view (e.g. to see things that we as maintainers do not see due to "operational blindness" or are just not aware of).

I don't know, but how do we think suggestions from the SIG security will be operationalized? Will they run through maintainer approval (like other important change requests do), or does the SIG have something like an "override" on that? In the former case, I see no problems with having someone "external" on board. In the latter case, how about having something like an "Adivsory council" or "Advisory board" without voting power within the SIG?

> | Name | Company | Email |

Reviewer: ```suggestion
| Name | Company | Email |
|--------|------|------|
```

# Argo SIG Security

Argo SIG Security is focused on improving security across the Argo project, promoting best practices, and ensuring a high level of security for end users.

## Meetings
Sig security meetings include public and non-public sections and are held twice monthly. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com).

## Security Policy and Reporting Issues
Argo Project security policy and instructions for reporting issues are available [here](https://github.com/argoproj/argoproj/blob/master/SECURITY.md).

## Members
| Name | Company | Email |
|--------|------|------|

> | Alexander Matyushentsev | Intuit | |

Reviewer 3: The org name needs to change here :)

Reviewer 4: Outtuit, Inkuity
